app/src/main/java/helium314/keyboard/keyboard/clipboard/ClipboardHistoryView.kt

```diff
@@ -240,6 +242,22 @@ class ClipboardHistoryView @JvmOverloads constructor(
         }
     }
 
+    override fun onLongClick(view: View): Boolean {
+        val tag = view.tag
+        if (tag is ToolbarKey) {
+            val longClickCode = getCodeForToolbarKeyLongClick(tag)
+            if (longClickCode != KeyCode.UNSPECIFIED) {
+                keyboardActionListener?.onCodeInput(
+                    longClickCode,
+                    Constants.SUGGESTION_STRIP_COORDINATE,
+                    Constants.SUGGESTION_STRIP_COORDINATE,
+                    false
+                )
+            }
+        }
+        return true
```

Comment:
I think you should only return true if the key is a toolbar key, or if the toolbar key actually has a specified key code.
[Documentation](https://developer.android.com/reference/android/view/View.OnLongClickListener#onLongClick(android.view.View)) states that the return value should be "true if the callback consumed the long click, false otherwise."

Or did you already try it and it caused issues?

Reply:
i don't think this will have an effect because there are no views with a fallback long click function but i can change it just in case you decide to add one

Reply:
Thanks!
I prefer to stick with behaving as documented whenever (easily) possible, makes like easier when changing things later.